packages/stargate/scripts/define-proto.sh

```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
 set -o errexit -o nounset -o pipefail
-command -v shellcheck > /dev/null && shellcheck "$0"
+command -v shellcheck >/dev/null && shellcheck "$0"
 
 TMP_DIR="./tmp"
 JS_SOURCE_FILE="$TMP_DIR/codecimpl.js"
 DEFINITIONS_FILE="$TMP_DIR/codecimpl.d.ts"
 OUTPUT_DIR="./src/codec/generated/"
 
-
```

Comment:
Whiy this diff? This is already done in https://github.com/cosmos/cosmjs/pull/567

Reply:
Ah, need to rebase